cmd/krel/templates/latest/metadata.yaml

```diff
@@ -69,6 +69,11 @@ kubeadm:
         versionConstraint: ">= 1.2.0"
       - name: cri-tools
         versionConstraint: ">= 1.28.0"
+  - versionConstraint: ">= 1.30.0"
```

Comment:
I want to double-check if this is going to include prereleases, because if it does, packages for prereleases would be broken.

Reply:
It doesn't match prereleases: https://go.dev/play/p/6akGpze_CeL